docs/openapi/schemas.yaml

```diff
@@ -1475,3 +1475,48 @@ BrokenBacklink:
     url_suggested:
       description: The URL suggested for redirecting from the broken backlink.
       $ref: '#/URL'
+ImportAssistantRequest:
+  type: object
+  required:
+    - url
+    - command
+  properties:
+    url:
+      description: The URL to analyze
+      $ref: '#/URL'
+    command:
+      description: Assistant command to execute.
+      type: string
+    prompt:
+      description: Optional prompt to provide context for the analysis.
+      type: string
+    options:
+      description: Optional configuration options required by the command.
+      type: object
+  example:
+    - url: 'https://example.com/en/page1'
+      command: 'start'
+    - url: 'https://example.com/en/page2'
+      command: 'cleanup'
+      prompt: 'breadcrumbs'
```

Comment:
This example suggests the ImportAssistantRequest is actually an `array`, instead of an `object`:

<img width="482" alt="image" src="https://github.com/user-attachments/assets/eb10c4de-2357-4db0-8f6a-3912ad7b659e">

Reply:
Oops, that is incorrect. I was attmepting to provide two different request examples.